src/features/layout/TopBar.tsx

```diff
@@ -38,14 +28,25 @@ export default function TopBar() {
   return (
     <header className={classes.topbar}>
       <div className={classes.topbar_item}>
-        <HamburgerMenu />
-      </div>
-      <div className={classes.topbar_item}>
+        {/* TODO: replace with recntagular logo without tagline */}
```

Comment:
Whats the plan here? Does the asset not exist?

Reply:
I was able to find the asset and add it in. I'll remove the TODO.